Dockerfile

```diff
@@ -184,6 +185,11 @@ RUN --mount=target=/root/.cache,type=cache \
   xx-verify --static /out/containerd-stargz-grpc && \
   xx-verify --static /out/ctr-remote
 
+FROM gobuild-base AS nydus
+ARG NYDUS_VERSION
+RUN wget https://github.com/dragonflyoss/image-service/releases/download/$NYDUS_VERSION/nydus-static-$NYDUS_VERSION-x86_64.tgz
```

Comment:
Hi @sipsma, thanks for the feedback. We are trying to release an arm64 version for nydus first so that the tests can pass on most arm architecture machines.

Reply:
@sipsma Added os/arch check, since nydus has only been tested in production on `linux/amd64` and `linux/arm64` machines, it is currently only guaranteed to be available on these two platforms, in the future we will add support for other linux archs or macOS, etc, thanks.